pinot-core/src/main/java/org/apache/pinot/core/io/reader/impl/v1/BaseChunkSingleValueReader.java

```diff
@@ -75,7 +67,7 @@ public BaseChunkSingleValueReader(PinotDataBuffer pinotDataBuffer) {
     headerOffset += Integer.BYTES;
 
     int dataHeaderStart = headerOffset;
-    if (_version > 1) {
+    if (version > 1) {
```

Comment:
The version was made into a member variable so that when we add or bump the version it is available to derived classes. Not sure why you removed it.

Recently we made a version bump and it was painful  and re-factoring had to be done. Some background in PR #5285

Reply:
It is not used anywhere, and IDE will have warning and can auto convert it to local.
I don't see why this can affect version bump as long as you have the version information inside the data buffer.